elements/urth-core-bind/urth-core-bind.html

```diff
@@ -74,12 +76,19 @@
                  */
                 channel: {
                     type: String,
-                    value: 'default',
```

Comment:
Yes, channel element has the `default` value so when `urth-core-bind` creates the `urth-core-channel` element it will be set.